### ISSUE TITLE
Mutagen complains about not being the right version

### ISSUE THREAD

**Issue by erfanimani** [davidalger/warden#95]
Using OSX Catalina, fresh install of warden.

```
warden sync start
> Mutagen could not be found; attempting install via brew.
...
==> Summary
🍺  /usr/local/Cellar/mutagen/0.11.0: 5 files, 78.0MB, built in 1 minute 35 seconds
Mutagen version 0.10.3 or greater is required (version 0.11.0 is installed).

Please update Mutagen:

  brew upgrade havoc-io/mutagen/mutagen
```

and again

```
warden sync start
Mutagen version 0.10.3 or greater is required (version 0.11.0 is installed).

Please update Mutagen:

  brew upgrade havoc-io/mutagen/mutagen
```

When running brew upgrade
```
$ brew upgrade havoc-io/mutagen/mutagen
Warning: havoc-io/mutagen/mutagen 0.11.0 already installed
```

It works when installing mutagen manually with `brew install mutagen@0.10` (after unlinking the previous version).

**Comment by davidalger** [comment]
@erfanimani Thanks for reporting this issue. Sounds like I had some faulty logic in here for the version check and @blakesaunders has submitted PR #94 to resolve this issue. Just returned from a week of business travel, but I'll get it reviewed and cut a new release as soon as I can.

**Comment by davidalger** [comment]
Published in Warden 0.2.3 release.

**Comment by erfanimani** [comment]
Much appreciated!
